_docs/welcome/feature-stages.md

```diff
@@ -30,28 +30,18 @@ within the product, not to the product as a whole. Here is a high level descript
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Config / APIs
```

Comment:
Seems fine to remove these, not really well specified enough as features.

Reply:
+1

Reply:
agreed.